repository/s.json

```diff
@@ -3469,12 +3469,17 @@
 		},
 		{
 			"name": "Squirrel",
-			"details": "https://github.com/iAmRoland/Sublime-Squirrel",
-			"author": ["micheg", "Roland Piirsoo"],
+			"details": "https://github.com/Enduriel/Sublime-Squirrel",
+			"author": ["Enduriel", "micheg", "Roland Piirsoo"],
 			"labels": ["language", "syntax", "snippets", "completions", "squirrel"],
 			"releases": [
 				{
-					"sublime_text": "*",
+					"base": "https://github.com/iAmRoland/Sublime-Squirrel",
+					"sublime_text": "<3092",
+					"branch": "master"
```

Comment:
Actually, ST < 3092 is not supported either, there is no need to take those versions into account here. Especially not via branch based releases.